### DISCUSSION TITLE
Collapsible Sidebar Menu & Navigation Drawer

### DISCUSSION THREAD

**Discussion by jlb6907** [slint-ui/slint#2266] [Q&A] [answered]
Hi,

Is it possible to have an example of a Collapsible Sidebar Menu & Navigation Drawer with slint ?
(Like [this](https://www.youtube.com/watch?v=eT9wTf6HFS8))

Thanks a lot.

**Comment by ogoffart** [comment, marked as answer]
Sure it is possible.

Does [this](https://slint-ui.com/releases/0.3.5/editor/?load_url=https://gist.githubusercontent.com/ogoffart/bd8a1f6f6582d34af6b9ce5fbb9b7b05/raw/1f6d4e6dd1451ef9dbe6564e6f42fcddcb798d98/side-menu.slint) help?

Edit: the URL was too big: this one is small enough: https://slint-ui.com/releases/0.3.5/editor/?load_url=https://gist.githubusercontent.com/ogoffart/bd8a1f6f6582d34af6b9ce5fbb9b7b05/raw/1f6d4e6dd1451ef9dbe6564e6f42fcddcb798d98/side-menu.slint

**Reply by jlb6907** [reply]
Thanks a lot.

**Reply by iincity** [reply]
use slint-viewer side-menu.slint,it can not show complete
![image](https://github.com/slint-ui/slint/assets/5435797/e749a255-53a6-4713-9272-35e6f1d766b0)
